x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryTranslatorTests.java

```diff
@@ -516,7 +511,8 @@ public void testTranslateRoundWithOneParameter() {
         Round groupingRound = (Round) ((Aggregate) p).groupings().get(0);
         assertEquals(1, groupingRound.children().size());
 
-        QueryTranslator.GroupingContext groupingContext = QueryTranslator.groupBy(((Aggregate) p).groupings());
+        QueryTranslator.GroupingContext groupingContext = TestUtils.withContext(TestUtils.TEST_CFG, resolution,
```

Comment:
You could probably create a private method that should just receive the runnable as parameter, and in turn the method to wrap the runnable with the `withContext(TestUtils.TEST_CFG, resolution...` call?